src/components/UriViewer/UriViewer.js

```diff
@@ -41,6 +42,16 @@ export const UriViewer = _.flow(
         })
         const metadata = await loadObject(googleProject, bucket, name)
         setMetadata(metadata)
+      } else if (isAzureUri(uri)) {
+        // assumption is made that container name guid in uri always matches the workspace Id guid it is present in
+        const workspaceId = uri.split('/')[3].replace('sc-', '')
```

Comment:
thought of doing this with regex for matching on the guid/sc-guid but in theory someone could name a folder or file a similar way and then we would get the wrong workspace id. would take any other ideas/suggestions gladly!